src/App.scss

```diff
@@ -1 +1,44 @@
-// styles go here
+ul {
```

Comment:
It's always good to use classes to get access to the props to avoid specificity problems.